apps/src/templates/instructions/TopInstructionsCSF.jsx

```diff
@@ -140,10 +143,15 @@ var TopInstructions = React.createClass({
    * @returns {number} The minimum height of the top instructions (which is just
    * the height of the little icon and the height of the resizer if we're not
    * collapsed
-
    */
   getMinHeight() {
-    return $(ReactDOM.findDOMNode(this.refs.icon)).outerHeight(true) +
+    const buttonHeight = $(ReactDOM.findDOMNode(this.refs.collapser)).outerHeight(true);
+    const minIconHeight = this.refs.icon ?
+      $(ReactDOM.findDOMNode(this.refs.icon)).outerHeight(true) : 0;
+    const minInstructionsHeight = this.props.collapsed ?
+      $(ReactDOM.findDOMNode(this.refs.instructions)).outerHeight(true) : 0;
+
+    return Math.max(buttonHeight, minIconHeight, minInstructionsHeight) +
```

Comment:
Were having some issues where in the collapsed state, instructions and/or the icon were being cut off. New heuristic is that in collapsed mode, our min height needs to fit all three of icon, button, and instructions.